### DISCUSSION TITLE
Docker Instructions

### DISCUSSION THREAD

**Discussion by madman2021** [postalserver/postal#1435] [Help with using Postal]
I've just noticed one of the latest commit contains a DockerFile.

I've build the docker file and then entered it in an interactive shell, but I can't get postal running with `.postal run` it complains about line 23: procodile:

I'd assume it also needs config piping in for database etc, I didn't try anything else after the error above, I assume since this is in master it's ready for use?

I've looked around for any examples / instructions and can't find any, am I missing something or is this not ready for prime time yet.

**Comment by willpower232** [comment]
As you say, it is a recent addition.

Its primary purpose is for kubernetes high availability (#974) so the command given to the docker should be one of the processes listed in Procfile so that each process can be separated within ones cluster.

You may be interested in some messing around that I have done in https://github.com/SynergiTech/postal/commit/911a5a39d8e8839367b979c03d63c81384cf13f5
